src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const element = document.querySelectorAll('.population');
+const elementTotal = document.querySelector('.total');
+const elementAverage = document.querySelector('.average');
+const text = [...element].map(item => item.innerText);
+let numbersStr = '';
+const convertString = text.join(' ');
+
+for (let i = 0; i < convertString.length; i++) {
+  numbersStr += convertString[i].replace(',', '');
+}
+
+const arr = numbersStr.split(' ');
+const numbersArr = [];
+
+for (let i = 0; i < arr.length; i++) {
+  numbersArr.push(+arr[i]);
+}
+
+const total = numbersArr.reduce((x, y) => {
+  return x + y;
+}, 0);
+
+const average = numbersArr.reduce((x, y) => {
+  return Math.round((x + y) / numbersArr.length);
+}, 0);
```

Comment:
redundant method (in fact, this is another additional loop, which means additional calculation). Try to optimize your solution. Create variable and increment it in 19 line in previous loop